pkg/lib/config/feature.go

```diff
@@ -88,3 +88,7 @@ func ParseFeatureConfig(inputYAML []byte) (*FeatureConfig, error) {
 
 	return &config, nil
 }
+
+func NewEffectiveDefaultFeatureConfig() *FeatureConfig {
+	return &FeatureConfig{}
```

Comment:
I think we should populate the struct recursively so that we will not encounter nil pointer exception when we access the fields. I remember we have done the same to authgear.yaml.
That should be done for both `NewEffectiveDefaultFeatureConfig` and `ParseFeatureConfig`

Reply:
I did it in the later commit, by using the `setFieldDefaults` function like `authgear.yaml`.